src/endpoints/sources.js

```diff
@@ -620,8 +624,23 @@ function copyHeaders(headerSource, headerNames) {
  * @returns {String} Path for {@link Source#dataset} or {@link Source#narrative}
  */
 
+/* Confused about the duplication below?  It's the documented way to handle
+ * overloaded (e.g. arity-dependent) function signatures.¹  Note that it relies
+ * on the "nestled" or "cuddled" end and start comment markers.
+ *   -trs, 16 June 2022
+ *
+ * ¹ https://github.com/jsdoc/jsdoc/issues/1017
+ */
```

Comment:
I'm going to take my time to digest this 🤔

Reply:
lmk if you need any digestifs.